Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1213,6 +1879,10 @@ def send_message():
             investigation: dict = demisto.investigation()
             investigation_id = investigation.get('id', '')
         channel_id = get_channel_id(channel_name, team_aad_id, investigation_id)
+        if get_channel_type(channel_id, team_aad_id) != 'standard':
+            raise ValueError('Posting a message or adaptive card to a private\shared channel is currently '
+                             'not supported.')
+
```

Comment:
Let's move to an extra function lines 1872-1885
it will look like:

```
    if channel_name:
        channel_id = new_func(message_type, channel_name)
    elif team_member:
```